device/transport/amqp/test/_amqp_test.js

```diff
@@ -97,14 +94,15 @@ describe('Amqp', function () {
     it('does not subscribe to the newTokenAvailable event if the authenticationProvider is x509', function () {
       fakeX509AuthenticationProvider.on = sinon.stub();
       var amqp = new Amqp(fakeX509AuthenticationProvider, fakeBaseClient);
+      void(amqp);
```

Comment:
>void(amqp); [](start = 6, length = 11)

huh. why?

Reply:
jshint.  If we don't do this, it complains about not using the value.  If we don't assign it to an object, it complains for some other reason.

---
In reply to: [180590352](https://github.com/Azure/azure-iot-sdk-node/pull/271#discussion_r180590352) [](ancestors = 180590352)